plasmapy/plasma/sources/plasmablob.py

```diff
@@ -134,7 +134,7 @@ def coupling(self):
         kinetic energy.
         """
         couple = coupling_parameter(
-            self.T_e, self.n_e, (self.particle, self.particle), self.Z
+            self.T_e, self.n_e, (self.particle, self.particle), self.Z*u.dimensionless_unscaled
```

Comment:
Yes! I may mixed up the two commits. sorry I'm new to git :/.  How do you think is the correct work flow in order to contribute to the code. I mean how many files edited per commit and how many  per pull and revision. I don't know If I explain myself. I will look at some git tutorial nonetheless. Thanks!

Reply:
No worries!  My favorite description of the workflow is from [Astropy's dev guide](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#workflow).

> I mean how many files edited per commit and how many per pull and revision.

That's a good question!  When making a commit or pull request, it helps to make sure that the changes are related to each other.  If a commit makes two separate changes, it's harder if you want to reverse one of the changes but not the other. It's also really helpful to write pull requests that are ≲400 lines long since they can be reviewed more completely.  I often try to limit pull requests to a single function or class, though sometimes larger pull requests are necessary.

Thank you again for contributing!